pyrsh.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+import sys
+from contextlib import contextmanager
+import time
+
+def arguments():
+    parser = argparse.ArgumentParser(description='Remote command execution via ssh')
+    parser.add_argument('host', help='Hostname', type=str)
+    parser.add_argument('user', help='Your user', type=str)
+    parser.add_argument('password', help='Your password', type=str)
+    parser.add_argument('cmd', help='Your cmd', type=str)
+    parser.add_argument('port', nargs='?', default=22, help='Enter your port, default port=22', type=int)
+    result = parser.parse_args()
+    return result
+    
+def response(channel):
+    while not channel.recv_ready():
+        time.sleep(0.1)
+    stdout = ''
+    while channel.recv_ready():
+        stdout += channel.recv(1024)
+    return stdout
+
+@contextmanager
+def ssh(args):
+    connect = paramiko.SSHClient()
+    connect.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    connect.connect(hostname=args.host, port=args.port, username=args.user, password=args.password)
+    yield connect
+    connect.close()
+
+def main(args):
+    with ssh(args) as connect:
```

Comment:
@GitSergeyGit Ok, how about introducing different ssh client support? I'm not saying to get rid of `paramiko`, but if there is `ssh` command in `PATH`, lets try using it instead of some third party lib. This way we will work out out own some additional layer, to hide specific implementation details.

Here is usage of an ssh cmdline to run command remotely
`ssh user1@server1 'df -H'`